src/requirements.txt

```diff
@@ -1,6 +1,7 @@
 flake8==3.7.8
 Flask==1.1.1
 Flask-Cors==3.0.8
+Flask-HAL==1.0.4
 -e git+https://github.com/Amsterdam/GOB-Core.git@v0.8.10c#egg=gobcore
 pytest==5.1.2
 pytest-cov==2.7.1
```

Comment:
I suggest to remove empty test files

Reply:
Yes, they weren't supposed to come along